web/jsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This enables VSCode IntelliSense for Cypress:

<img width="948" alt="Screenshot 2022-06-20 at 07 54 01" src="https://user-images.githubusercontent.com/3296643/174534239-bd264c9c-bb74-47f1-9a8c-8a2c9aae2087.png">